src/main/java/org/scalameter/japi/annotation/scopes.java

```diff
@@ -0,0 +1,20 @@
+package org.scalameter.japi.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/** Allows to set custom contexts for scopes.
+ *
+ *  It should consists of @scopeCtx that each points to a single scope and context.
+ *
+ *  Note that it should be placed over a class
+ *  that extends `JBench` and contains benchmark methods.
```

Comment:
I like the idea here.

Please include a comment of how the context is resolved for multiple colliding scopes.

Also, please add an example here.